src/app/layouts/base-layout/components/partnered/partnered.scss

```diff
@@ -12,25 +12,31 @@
     }
 
     & .partners {
-      @include media-tablet {
-        flex-direction: column;
-        width: 100%;
-
-        & > img {
-          margin-top: 24px;
-        }
-      }
-
-      overflow: hidden;
       display: flex;
-      flex-flow: row nowrap;
-      place-content: center space-between;
+      flex-flow: row wrap;
       align-items: center;
       align-self: center;
+      justify-content: space-between;
+
+      width: 100%;
+      max-width: 500px;
+      padding: 16px;
 
-      width: 500px;
-      height: min-content;
-      padding: 16px 16px 16px 16px;
+      @include media-tablet {
```

Comment:
move media to the top, please

Reply:
Sure, I will do the thing. 

Please, could you give me a reference to documentation on why we should do this? Is there any benefit?